layouts/partials/header.html

```diff
@@ -25,10 +25,12 @@
               <span class="usa-banner__button-text">{{ $translation.homepage.banner__banner_action }}</span>
             </button>
           </div>
-
+          
+        {{ if ne .Type "nvrf" }}
           <div class="switcher-desktop">
           {{ partial "language-switcher.html" . }}
           </div>
+        {{ end }}
```

Comment:
Let's remove the condition here. We want the language switcher to show up when we do create a translation.

Reply:
Ah okay, the ticket specified to hide the lang selector and search. I'm going to update the ticket